gateway/queue/Cargo.toml

```diff
@@ -15,9 +15,9 @@ repository = "https://github.com/twilight-rs/twilight.git"
 version = "0.6.0"
 
 [dependencies]
-tokio = { default-features = false, features = ["net", "rt-multi-thread", "sync"], version = "1.0" }
+tokio = { default-features = false, features = ["rt", "sync", "time"], version = "1.0" }
```

Comment:
Did we mistakenly enable `net` before? Did `net` imply `time`?

Reply:
tokio features generally don't depend on each others. `net` was unused and `time` is used for sleeping

Reply:
This might be a dumb question but how did we have a `tokio::time` import without the feature enabled before?

Reply:
other crates required it. the new feature resolver (coming with rust 2021) would probably not allow it

Reply:
I suppose that's the answer to the rest of the new features